cli.js

```diff
@@ -22,6 +22,11 @@ if (!args.dir || (!args.all && (!args.platform || !args.arch))) {
   process.exit(1)
 }
 
+// minimist doesn't support to specify several types (string|boolean)
```

Comment:
Fixed, thanks.







> On Mar 1, 2016, at 8:22 AM, Mark Lee <notifications@github.com> wrote:
> 
> 
> In cli.js
> :
> 
>> @@ -22,6 +22,11 @@ if (!args.dir || (!args.all && (!args.platform || !args.arch))) {
>>    process.exit(1)
>>  }
>>  
>> +// minimist doesn't support to specify several types (string|boolean)
> It would read better as:
> 
> // minimist doesn't support multiple types for a single argument (in this case, `String` or `false`)
> 
> —
> Reply to this email directly or view it on GitHub
> .<embedded image>
> 
>